src/Paket.Core/Common/ProcessHelper.fs

```diff
@@ -26,7 +27,9 @@ let tryFindFile dirs file =
                else
                    let fi = FileInfo(Path.Combine(dir.FullName,file))
                    if fi.Exists then fi.FullName
-                   else "")
+                   else ""
+            with
+            | _ -> "")
```

Comment:
Should we print the exception message and the bad path?

Reply:
Maybe always print something like "Error when searching \<file\> in \<path\>" and print exception message only in verbose mode?
Or print everything only in verbose?

Reply:
I'd print that as a warning, otherwise people might not even be aware that they have something invalid in their `PATH`.

Reply:
I think it can be confusing to see warn about some strange path after hit `paket restore` in random repository.
And in `tryFindFile` we dont know source of `dirs` parameter.
So i'm inclined to verbose-only trace.